src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private static final double CONSTANT = 0.5;
+    private double baseUpper;
+    private double baseLower;
+    private double height;
+
+    public IsoscelesTrapezoid(Color color, double baseLower,
+                              double baseUpper, double height) {
+        super(color);
+        this.baseLower = baseLower;
+        this.baseUpper = baseUpper;
+        this.height = height;
+    }
+
+    @Override
+    public double areaCalculator() {
+        return (baseLower + baseUpper) * CONSTANT * height;
```

Comment:
```suggestion
        return (baseLower + baseUpper) * 0.5 * height;
```
the name is not informative